components/backup/src/endpoint.rs

```diff
@@ -1141,8 +1144,9 @@ impl<E: Engine, R: RegionInfoProvider + Clone + 'static> Endpoint<E, R> {
             }
         };
         let backend = Arc::<dyn ExternalStorage>::from(backend);
-        let concurrency = self.config_manager.0.read().unwrap().num_threads;
-        self.pool.borrow_mut().adjust_with(concurrency);
+        let cur_num_threads = self.config_manager.0.read().unwrap().num_threads;
+        self.pool.borrow_mut().adjust_with(cur_num_threads);
+        let concurrency = cur_num_threads * 3;
```

Comment:
Please add comment to explain why multiple 3 here.

Reply:
Just for test